miso-web/src/main/resources/internal/miso.properties

```diff
@@ -59,6 +59,9 @@ miso.notification.interop.enabled:false
 miso.notification.server.host:your.notification.server
 miso.notification.server.port:8787
 
+# A white-space delimited list of run scanner instances to poll
+miso.runscanner.urls:http://localhost:8080/runscanner
```

Comment:
Should this be added to the `miso.properties` in `resources/external` as well? If so, please make some note in the PR (title or commit message) that this is a configuration change so it can be easily found in the release notes.